src/main/java/core/basesyntax/RandoM.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandoM extends Enum {
+    public Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
Calling 'Color.values()' inside a loop or repeatedly in a method can be inefficient because it creates a new array every time it is called. Consider storing the result of 'Color.values()' in a class-level constant array and using that instead.